server/community_test.go

```diff
@@ -93,3 +93,214 @@ func TestPostPRWelcomeMessage(t *testing.T) {
 		})
 	}
 }
+
+func TestAssignCommunityLabels(t *testing.T) {
+	pr := &model.PullRequest{
+		RepoOwner: "owner",
+		RepoName:  "repoName",
+		Number:    123,
+		Username:  "foo",
+	}
+
+	t.Run("Org Member", func(t *testing.T) {
+		repo := &Repository{
+			Owner:          "owner",
+			Name:           "repoName",
+			GreetingLabels: []string{"hey", "hello"},
+		}
+
+		s := &Server{
+			Config: &Config{
+				Repositories: []*Repository{repo},
+			},
+			OrgMembers: []string{"foo"},
+		}
+		assert.NoError(t, s.assignCommunityLabels(context.Background(), pr, repo))
```

Comment:
I'm guessing the absence of `EXPECT` assertions here is sufficient to verify that we /aren't/ doing anything for org members in this case. Is there any way with gomock to make that explicit for clarity in the test case itself?

Reply:
I'll explicitly add that, but since the `server` doesn't have a githubclient configured it would fail